### ISSUE TITLE
Spellchecker Language not saved correctly

### ISSUE THREAD

**Issue by vantezzen** [getferdi/ferdi#1016]
**Bug description:**

I am currently experiencing a bug on Mac where changing any setting will take a very long time (5-10 seconds). While investigating this, I noticed that my `settings.json` is growing in size with every change I make, now reaching almost 300MB for the simple JSON file:

<img width="499" alt="Screenshot 2020-10-11 at 17 22 47" src="https://user-images.githubusercontent.com/10333196/95682603-63391100-0be6-11eb-88c8-ecf6af36785c.png">

Looking at the content of the file, everything seems normal except the spellchecker language (excerpt from the file - the image should do as trying to copy anything from this file results in my editor crashing 😅):

<img width="938" alt="Screenshot 2020-10-11 at 17 24 44" src="https://user-images.githubusercontent.com/10333196/95682641-a8f5d980-0be6-11eb-9c3f-2c65279ba947.png">

For reference, this is what this option should look like:
```
"spellcheckerLanguage": "[\"en-US\"]",
```

This is what it looks like after one save:
```
"spellcheckerLanguage": "\"[\\\"en-US\\\"]\"",
```

It looks like the escaping backslashes in the JSON string are getting escaped themselves, which in turn also gets escaped etc..

**Steps to reproduce:**

1. Open settings in Ferdi
2. I don't know if there are specific options that trigger this or if it happens with all options - I switched some options under "Appearance"
3. Watch the settings.json grow in size

**Environment:**

- Operating System:  macOS
- Ferdi Version:  Current develop branch
- Server:  Without server

**Comment by alindl** [comment]
This is still an issue I noticed myself. As you said, you can't even open the file and delete that line. It makes Ferdi impossible to use as soon as you go into the settings. I was wondering, why opening settings is getting slower over time 😅

My workaround was to close Ferdi and delete that line using `sed -i '15d' ~/.config/Ferdi/config/settings.json`, but change the line number `15` according to where that line is in your own file. Be careful though, you are just blindly deleting a line here, make sure this is the correct line.

**Comment by kris7t** [comment]
This was fixed in #1314
